### PR TITLE
Feature population heatmaps

### LINKED ISSUE #362: Implement population heat map layer

### PR DESCRIPTION
**Pull request by vaaralav**
Fixes #362

### CODE REVIEW

File: src/data-visualization.coffee [hunk 1]
```diff
@@ -0,0 +1,29 @@
+define []
+, () ->
+    class DataVisualization
+        # Data layers as in name: map_id
+        HEATMAP_DATASETS =
```
Comment (vaaralav) [inline src/data-visualization.coffee]: TODO: Insert actual heat map IDs here

**Comment by tituomin** [comment]
Thanks! A couple of fixes needed:
 - make sure only one layer is inserted at a time. currently if you select multiple layers in succession using the radio buttons, they are inserted on top of each other. this probably was already a problem in the initial example implementation
 - add a selection for removing any pop heat map layers: "ei näkyvissä / not selected" or similar

**Comment by tituomin** [comment]
Also:
- remove the dummy "show me some stats"

**Comment by tituomin** [comment]
Also:
- maybe make the header dynamic, similar to the map background (replace the text "väestötiedot" with the selected layer), and automatically collapse the element on selection (otherwise a lot of the actual map visualisation is hidden behind the element)

**Pull request merged**